praw/models/helpers.py

```diff
@@ -4,13 +4,127 @@
 
 from ..const import API_PATH
 from .base import PRAWBase
+from .reddit.draft import Draft
 from .reddit.live import LiveThread
 from .reddit.multi import Multireddit, Subreddit
 
 if TYPE_CHECKING:  # pragma: no cover
     import praw
 
 
+class DraftHelper(PRAWBase):
+    """Provide a set of functions to interact with Drafts."""
+
+    def __call__(
+        self, draft_id: Optional[str] = None
+    ) -> Union[List["praw.models.Draft"], "praw.models.Draft"]:
+        """Return a list of the currently authenticated user's Drafts.
+
+        :param draft_id: When provided, return :class:`.Draft` instance (default:
+            ``None``).
+
+        :returns: A :class:`.Draft` instance if ``draft_id`` is provided. Otherwise, a
+            list of :class:`.Draft` objects.
+
+        .. note::
+
+            Drafts fetched using a specific draft ID are lazily loaded, so you might
+            have to access an attribute to get all the expected attributes.
+
+        This method is to be used to fetch a specific draft, like so:
+
+        .. code-block:: python
+
+            draft_id = "124862bc-e1e9-11eb-aa4f-e68667a77cbb"
+            draft = reddit.drafts(draft_id)
+            print(draft)
+
+        """
+        if isinstance(draft_id, str):
+            return [Draft(self._reddit, id=draft_id)]
+        return self._draft_list
+
+    @property
```

Comment:
Is there an advantage of using the `property` on the `_draft_list`?
Are we benefiting from this or is this a property that is followed in praw?

Reply:
It is similar to how rules are implemented.

Reply:
Since this is a "private" method, I'd recommend against making a property as I think there is little value in doing so. If the same is true for rules then we should reconsider having it a property.